packages/react/src/components/Autofill/Autofill.tsx

```diff
@@ -174,8 +174,8 @@ export class Autofill extends React.Component<IAutofillProps, IAutofillState> im
 
     if (inel && inel.selectionStart !== this.value.length) {
       return {
-        start: inel.selectionStart || inel.value.length,
-        end: inel.selectionEnd || inel.value.length,
+        start: inel.selectionStart !== null ? inel.selectionStart : inel.value.length,
+        end: inel.selectionEnd !== null ? inel.selectionEnd : inel.value.length,
```

Comment:
Generally null and undefined should be handled the same way (unless there's a specific reason they need to be different in this case). The easiest way to do this is with the [nullish coalescing operator `??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).
```suggestion
        start: inel.selectionStart ?? inel.value.length,
        end: inel.selectionEnd ?? inel.value.length,
```